src/inventorymanager.cpp

```diff
@@ -523,6 +523,43 @@ void IMoveAction::apply(InventoryManager *mgr, ServerActiveObject *player, IGame
 		}
 	}
 
+	InventoryLocation from_loc;
+	if(from_inv.type == InventoryLocation::NODEMETA)
```

Comment:
Should i edit my patch immediately to this pull accepted, or this is just a note for a future?